### PR TITLE
refactor(docs): rename doc to docs

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR renames the `/doc` directory to `/docs`

## Why?

In order for GitHub to automatically recognize our `CONTRIBUTING.md` file, it must be in one of the following places:
- the root of the repository
- the `.github` folder
- the `docs` folder

(Source: [About default community health files](https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file#about-default-community-health-files))

As you can see on our [Community page](https://github.com/cdr/code-server/community), GitHub doesn't recognize that we have a `CONTRIBUTING.md` file.

image

I'd like to change that so we're following [GitHub's guidelines for healthy contributions](https://docs.github.com/en/github/building-a-strong-community/setting-up-your-project-for-healthy-contributions).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,4 +14,5 @@ node-*
 /plugins
 /lib/coder-cloud-agent
 .home
-coverage
\ No newline at end of file
+coverage
+**/.DS_Store
```
Comment (oxy) [inline .gitignore:18]: Sneaky `.gitignore` change!
Not sure if you meant to include this in this PR.

**Review by oxy — COMMENTED** [review]
Looks good to me! Not sure about the `.gitignore` change.

File: .gitignore (hunk 1 above)
Comment (jsjoeio) [reply .gitignore:18]: Well, I keep running into that error (thanks macOS lol). So if it's okay, I would like to include it!

Comment (oxy) [reply .gitignore:18]: Sounds good then!

**Review by oxy — APPROVED** [review]
Looks good!

**Pull request merged**